### PR TITLE
Remove blob

### PR DESCRIPTION
**Pull request by cwarnerdev**

### CODE REVIEW

File: tools/wasp-cli/chain/deploycontract.go [hunk 1]
```diff
@@ -36,7 +34,6 @@ func initDeployContractCmd() *cobra.Command {
 			client := cliclients.WaspClient(node)
 			vmtype := args[0]
 			name := args[1]
-			description := args[2]
 			initParams := util.EncodeParams(args[4:], chainID)
```
Comment (howjmay) [inline tools/wasp-cli/chain/deploycontract.go]: If we removed the description field, we may need to change the index of args here too?
```diff
- initParams := util.EncodeParams(args[4:], chainID)
+ initParams := util.EncodeParams(args[3:], chainID)
```

File: tools/wasp-cli/chain/deploycontract.go [hunk 2]
```diff
@@ -51,13 +48,8 @@ func initDeployContractCmd() *cobra.Command {
 				log.Check(err)
```
Comment (howjmay) [inline tools/wasp-cli/chain/deploycontract.go:48]: And 
```diff
- progHash, err = hashing.HashValueFromHex(args[3])
+ progHash, err = hashing.HashValueFromHex(args[2])
```

**Review by howjmay — CHANGES_REQUESTED** [review]

File: packages/origin/origin.go [hunk 3]
```diff
@@ -83,15 +82,13 @@ func InitChain(
...
-		blob.Contract,
 		accounts.Contract,
 		blocklog.Contract,
 		errors.Contract,
 		governance.Contract,
 		evm.Contract,
 	})
-	blob.NewStateWriter(blob.Contract.StateSubrealm(d)).SetInitialState()
-	accounts.NewStateWriter(v, accounts.Contract.StateSubrealm(d)).SetInitialState(originDeposit, baseTokenCoinInfo)
+	accounts.NewStateWriter(v, accounts.Contract.StateSubrealm(d)).SetInitialState(originDeposit)
```
Comment (dessaya) [inline packages/origin/origin.go]: `baseTokenCoinInfo` is needed here

File: packages/vm/execution/common.go [hunk 4]
```diff
@@ -14,7 +15,7 @@ func GetProgramBinary(ctx WaspContext, programHash hashing.HashValue) (vmtype st
 	if ok {
 		return vmtype, nil, nil
 	}
-	return ctx.LocateProgram(programHash)
+	return "", nil, fmt.Errorf("native processor not found for programHash")
```
Comment (dessaya) [inline packages/vm/execution/common.go]: This should probably be one of `coreerrors`

**Review by dessaya — APPROVED** [review]

File: packages/origin/origin.go (hunk 3 above)
Comment (cwarnerdev) [reply packages/origin/origin.go]: oops, messed that up with a merge conflict resolution. thanks!

**Pull request merged**